src/test/java/com/fundacionjala/pivotal/cucumber/hooks/GlobalHooks.java

```diff
@@ -0,0 +1,29 @@
+package com.fundacionjala.pivotal.cucumber.hooks;
+
+import com.fundacionjala.pivotal.exceptions.PropertiesInfoReadException;
+
+import cucumber.api.java.Before;
+
+import static com.fundacionjala.pivotal.framework.util.Constants.EMPTY_STRING;
+import static com.fundacionjala.pivotal.framework.util.PropertiesInfo.getInstance;
+
+/**
+ * Created by HENRRY on 18/07/2016.
+ */
+public class GlobalHooks {
+    private boolean BEFORE_ALL_FLAG = false;
+
+    @Before
+    public void beforeAll() {
+        if (!BEFORE_ALL_FLAG) {
+            if (EMPTY_STRING.equals(getInstance().getEmail()) || EMPTY_STRING.equals(getInstance().getApiToken()) || EMPTY_STRING.equals(getInstance().getPassword())) {
```

Comment:
You can use apache commons StringUtils isEmpty() or isNotEmpty().
http://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html